lib/factory/IntentRequestBuilder.ts

```diff
@@ -37,6 +39,18 @@ export class IntentRequestBuilder extends RequestBuilder {
         return this;
     }
 
+    public withSlotConfirmation(name : string, value : string, confirmationStatus : SlotConfirmationStatus) : IntentRequestBuilder {
```

Comment:
Can you please rewrite `withSlot` and `withEmptySlot` to use this method instead of duplicating the code?

Reply:
Update the withSlot and withEmptySlot to call into withSlotConfirmation and change order of parameters to make value optional.